Pipfile

```diff
@@ -4,37 +4,38 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-aiodns = "~=2.0.0"
-aiohttp-jinja2 = "~=1.2.0"
-aiohttp-session = {version = "~=2.9.0", extras = ["aioredis"]}
+aiohttp-jinja2 = "~=1.5"
+aiohttp-session = {version = "~=2.11.0", extras = ["aioredis"]}
 aiohttp-utils = "~=3.1.1"
-aiohttp = "~=3.7.4"
-aioredis = "~=1.3.1"
-babel = "==2.9.1"
-cchardet = "~=2.1.6"
+aiohttp = "~=3.8.1"
+aioredis = "~=2.0.1"
+babel = "==2.10.1"
+cchardet = "~=2.1.7"
 envparse = "~=0.2.0"
-invoke = "~=1.4.1"
-iso8601 = "~=0.1.12"
-python-json-logger = "~=0.1.11"
-structlog = "~=20.1.0"
-tenacity = "~=6.2.0"
-jinja2 = "~=2.11.2"
-importlib-metadata = "~=4.8.1"
-urllib3 = "~=1.26.5"
+invoke = "~=1.7.0"
+iso8601 = "~=1.0.2"
```

Comment:
I think `iso8601` is unused, tested locally and unit/cucumber tests pass without it